### PR TITLE
SFT-152 Fix Deprecration and Update Conan-related files

### PR DESCRIPTION
**Pull request by DatLam25**

### CODE REVIEW

File: EpsilonDashboardSetup.sh [hunk 1]
```diff
@@ -17,4 +17,9 @@ sudo apt-get install -y cmake \
 pip3 install --user conan
 echo 'export PATH=$PATH:~/.local/bin/' >> ~/.profile
 source ~/.profile
-conan remote add conan-dbely https://api.bintray.com/conan/dbely/conan
\ No newline at end of file
+conan remote add -f epsilon https://solarcar.jfrog.io/artifactory/api/conan/epsilon-conan
+read -p 'Conan Username: ' username
+read -sp 'Conan Password: ' password
```
Comment (liconway) [inline EpsilonDashboardSetup.sh:22]: Nice :+1:

File: conanfile.txt [hunk 2]
```diff
@@ -28,13 +25,13 @@ boost:without_math=True
 boost:without_mpi=True
 boost:without_program_options=True
 boost:without_python=True
+boost:without_nowide=True
 boost:without_random=True
 boost:without_regex=True
 boost:without_serialization=True
 boost:without_stacktrace=True
-boost:without_system=True
```
Comment (liconway) [inline conanfile.txt:35]: Seems we removed 3 blocked items here:
```
boost:without_container=True
boost:without_exception=True
boost:without_system=True
```

We also added 1:
```
boost:without_nowide=True
```

What's the reason for the change? Are we running a different version of Boost?

File: src/CommunicationLayer/CommDeviceControl/InternetConnectionService.cpp [hunk 3]
```diff
@@ -38,7 +48,8 @@ void InternetConnectionService::setupChannel()
 
     try
     {
-        channel_ = AmqpClient::Channel::Create(ipAddress_.toStdString(), port_);
+        //channel_ = AmqpClient::Channel::Create(ipAddress_.toStdString(), port_);
```
Comment (liconway) [inline src/CommunicationLayer/CommDeviceControl/InternetConnectionService.cpp:51]: I don't think there's any reason to leave this line commented out - if we need to revert, we should probably be looking at the commit history rather than comments.

File: src/CommunicationLayer/CommDeviceControl/InternetConnectionService.cpp [hunk 4]
```diff
@@ -25,6 +25,16 @@ InternetConnectionService::InternetConnectionService(
...
+
+    //Setting up openOpts
+    AmqpClient::Channel::OpenOpts::BasicAuth auth;
+    auth.password = "guest";
+    auth.username = "guest";
+    openOpts_.host = ipAddress_.toStdString();
+    openOpts_.port = port_;
+    openOpts_.auth = auth;
+    openOpts_.vhost = "/";
+    openOpts_.frame_max = 131072;
```
Comment (liconway) [inline src/CommunicationLayer/CommDeviceControl/InternetConnectionService.cpp:37]: Where'd this number come from? I'm assuming this code is a result of the `SimpleAmqpClient` upgrade, just hoping to verify that there's no behavioral change here.

**Review by liconway — COMMENTED** [review]
Looks really good overall, just want to verify a few things first

File: conanfile.txt (hunk 2 above)
Comment (DatLam25) [reply conanfile.txt:35]: The newer version of boost introduced nowide, which we do not use, hence the addition of without_nowide. On the other hand, atomic and chrono now requires system, contain and exception to work.

File: src/CommunicationLayer/CommDeviceControl/InternetConnectionService.cpp (hunk 4 above)
Comment (DatLam25) [reply src/CommunicationLayer/CommDeviceControl/InternetConnectionService.cpp:37]: There is no behavioral change, that is actually the default number of frame. In the deprecated code it is defaulted to the same value.

**Review by liconway — APPROVED** [review]
image
